docs/general_config.md

```diff
@@ -188,7 +188,7 @@ As part of the general configuration parameters several controls over reporting
 "logFrameInfo": true,
 "logPlayerActions": true,
 "logTrialResponse": true,
-"sessParamsToLog" : [],
+"sessParamsToLog" : ["frameRate", "frameDelay"],
```

Comment:
I think `sessParamsToLog` should be renamed to `sessionParametersToLog` to match the style of the rest of things about sessions and parameters. This is the only use of Parameters that I can find, so we could shorten to `Params`, but I prefer the couple extra characters for clarity.

Reply:
This should now be changed.